src/nodes/chunk_dispatch/chunk_dispatch.c

```diff
@@ -185,16 +187,86 @@ chunk_dispatch_plan_create(PlannerInfo *root, RelOptInfo *relopt, CustomPath *be
 		cscan->scan.plan.plan_rows += subplan->plan_rows;
 		cscan->scan.plan.plan_width += subplan->plan_width;
 	}
-
-	cscan->custom_private = list_make1_oid(cdpath->hypertable_relid);
+	cscan->custom_private = lappend(cscan->custom_private, makeInteger(cdpath->hypertable_relid));
```

Comment:
Why are we using Value nodes here instead of plain values?

Reply:
I cant use lappend_int() as it will create List of type T_IntList. And later when i append other values, i get assert.
Want to want is to be able to append relid + few other values. To achieve this i first convert relid as Integer node and save in the custom_private list.

How else can i store unsigned int and other values in a List ?